webpack-watcher.js

```diff
@@ -43,7 +44,27 @@ module.exports = (options) => {
    }
 
    function forkCompiler() {
-      return childProcess.fork(__dirname + '/webpack-compiler.js');
+      const forkOptions = getForkOptions();
+      const child = childProcess.fork(__dirname + '/webpack-compiler.js', forkOptions)
+      logOutput(child)
+      return child;
+   }
+
+   function getForkOptions() {
+      const output = options.logPath ? 'pipe' : 'inherit';
+      return {
+         stdio: ['inherit', output, output, 'ipc']
```

Comment:
I read the docs here, and I don't understand when/why this should be an array of 4 (as opposed to 3) things?https://nodejs.org/api/child_process.html#child_process_options_stdio

Reply:
> (for `.fork()`) When this option is provided, it overrides `silent`. If the array variant is used, it must contain exactly one item with value 'ipc' or an error will be thrown. For instance [0, 1, 2, 'ipc'].